File: src/main/java/com/drunkensoftball/api/game/repo/GameRepository.java
package com.drunkensoftball.api.game.repo;

import com.drunkensoftball.api.game.domain.Game;
import com.drunkensoftball.api.team.domain.Team;
import org.springframework.data.domain.Pageable;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.List;

public interface GameRepository extends JpaRepository<Game, Long> {

    Game findByTeamId(Long id);

}

File: src/main/kotlin/com/drunkensoftball/api/game/repo/GameRepository.kt
package com.drunkensoftball.api.game.repo

import com.drunkensoftball.api.game.domain.Game
import com.drunkensoftball.api.team.domain.Team
import org.springframework.data.domain.Pageable
import org.springframework.data.jpa.repository.JpaRepository

interface GameRepository : JpaRepository<Game, Long> {

    fun findByTeamId(id: Long?): Game
    fun findByUuid(uuid: String): Game

}
